roles/system_ldap/tasks/main.yml

```diff
@@ -1,97 +1,121 @@
 ---
-# tasks file for system_ldap
-#
-- name: system_ldap | Install sssd
-  ansible.builtin.apt:
-    name: "{{ item }}"
-    state: "present"
-  loop:
-    - "{{ sssd_pkg_list }}"
-
-- name: system_ldap | Remove unwanted packages
-  ansible.builtin.apt:
-    name: "{{ item }}"
-    state: "{{ sssd__unwanted_packages_state }}"
-  loop:
-    - "{{ sssd__unwanted_packages_list }}"
-
-- name: system_ldap | change hostname to match AD 
-  ansible.builtin.command: hostnamectl set-hostname {{ host_ad_name | default(omit)}}
+- name: System_ldap | change hostname to match AD
+  ansible.builtin.command: hostnamectl set-hostname {{ host_ad_name | default(omit) }}
+  changed_when: false
   when:
     - running_on_server
 
-- name: system_ldap | allow password authentication
+- name: System_ldap | allow password authentication
   ansible.builtin.lineinfile:
     path: /etc/ssh/sshd_config
     state: present
     regexp: '^PasswordAuthentication no'
     line: 'PasswordAuthentication yes'
 
-- name: system_ldap | allow users authentication
+- name: System_ldap | allow users authentication
   ansible.builtin.lineinfile:
     path: /etc/ssh/sshd_config
     state: present
     regexp: '^AllowUsers pulsys'
     line: '# AllowUsers pulsys'
   notify: restart sshd
```

Comment:
Earlier in this PR the `restart sshd` handler got deleted, so this `notify` parameter won't work.